app/format/format.tsx

```diff
@@ -116,6 +116,15 @@ export function formatCommitHash(commit: string) {
   return commit?.substring(0, 6);
 }
 
+export function formatWithCommas(num: Long) {
```

Comment:
In TypeScript, numbers usually have `number` type (instead of Long)

Reply:
It seems like the stat.totalNumBuilds and other variables have type Long because it errors when I use number

Reply:
Oh wow, the new code is much simpler. Nice!